key-rotator/manifest/manifest.go

```diff
@@ -132,7 +132,7 @@ type BatchSigningPublicKey struct {
 	// PKIX SubjectPublicKeyInfo structure. It must be an ECDSA P256 key.
 	PublicKey string `json:"public-key"`
 	// Expiration is the ISO 8601 encoded UTC date at which this key expires.
-	Expiration string `json:"expiration"`
+	Expiration string `json:"expiration,omitempty"`
```

Comment:
Did we discuss this in this or another PR? I'm not sure if it's a good idea to emit manifests without an expiration value.

Reply:
discussed in #1082, fixed there, merge from main did not pick up this change & I missed it :(